lib/ansible/module_utils/remote_management/spacewalk.py

```diff
@@ -0,0 +1,329 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd like to see integration tests to go along with these modules, if possible. They wouldn't likely be automated in CI/CD as username/password is required but it would make future changes easier to validate for possibly you, but the community at large if they were to contribute to it.